services/telemetry/config.default.yml

```
@@ -10,7 +10,7 @@ telemetry:
 
   - id: PMMServerSTTEnabled
     source: PMMDB_SELECT
-    query: stt->'stt_enabled' AS stt_enabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS stt
+    query: (CASE WHEN stt->'stt_disabled' = 'false' THEN 'true' ELSE 'false' END) AS stt_enabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS stt
```

stt_enabled no longer exist

---

can it be simplified like below?
```suggestion
    query: stt->'stt_disabled' != 'false' AS stt_enabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS stt
```